openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -347,10 +336,13 @@ def check_account_exists(username=None, email=None):
     """
     conflicts = []
 
-    if email is not None and User.objects.filter(email=email).exists():
+    try:
```

Comment:
I had to reuse this functionality elsewhere so I decided to drop it into a separate function, and decided to reuse it here.